packages/babel-helper-compilation-targets/test/browserslist-extends/browserslist-extends.spec.js

```diff
@@ -0,0 +1,44 @@
+import { join, dirname } from "path";
+import { promises } from "fs";
+import rimraf from "rimraf";
+import { fileURLToPath } from "url";
+import getTargets from "../..";
+
+const currentDir = dirname(fileURLToPath(import.meta.url));
+let mocked = [];
+
+async function mock(name, exports) {
+  const dir = join(
+    fileURLToPath(import.meta.url),
+    "..",
+    "..",
+    "..",
+    "node_modules",
+    name,
+  );
+  mocked.push(dir);
+  await promises.mkdir(dir, { recursive: true });
+  const content = "module.exports = " + JSON.stringify(exports);
+  await promises.writeFile(join(dir, "index.js"), content);
+}
+
+afterEach(() => {
+  mocked.map(dir => rimraf.sync(dir));
+  mocked = [];
+});
+
+it("pass env to configs used with extends", async () => {
+  await mock("@babel/browserslist-config", {
```

Comment:
You can create `node_modules/@babel/browserslist-config` (or `node_modules/test-custom-browserslist-config` so it works even if we do introduce our own browserslist config) under `packages/babel-helper-compilation-targets/test/browserslist-extends`, they should be considered part of the test fixtures and we don't have to dynamically generate them in the test.

Reply:
as browserslist uses code `require(require.resolve(name, { paths: ['.'] }))` I cannot put mock module inside test and had to put it somewhere in upper node_modules

Nevertheless, I managed the way to pass browserslist exact module on disk, so it looks better now and doesn't generate anything dynamically